Makefile

```diff
@@ -55,7 +55,7 @@ mock-test-%: clean-stores
 	trap "$(MAKE) -C mock_connect down" EXIT; \
 	CONNECT_CONTENT_BUILD_DIR="rsconnect-build-test" \
 	CONNECT_SERVER="http://$(HOSTNAME):3939" \
-	CONNECT_API_KEY="0123456789abcdef0123456789abcdef" \
+	CONNECT_API_KEY="21232f297a57a5a743894a0e4a801fc3" \
```

Comment:
did we rotate credentials at some point? Just making sure I'm following why this change is needed

Reply:
That was a static api key that mock_connect was expecting. I changed it to the insecure admin api key (`echo -n admin | md5sum`) since that's what we use in dev, but this is removed in my next PR anyway